kornia/geometry/nerf/camera_utils.py

```diff
@@ -0,0 +1,104 @@
+from typing import List
+
+import torch
+from defusedxml import lxml as ET
+
+from kornia.geometry.camera import PinholeCamera
+from kornia.geometry.nerf.types import Device
+
+
+class CameraParser:
```

Comment:
```suggestion
class CameraXmlParser:
```
but provide it somehow in `kornia/examples/nerf`

Reply:
`CameraParser` class has been removed. Parsing is done differently for Colmap `.txt` output files